### PR TITLE
fix: #1266 update create_user and update_user length in db

### PR DESCRIPTION
**Pull request by MCatherine1994**
refs: #1266

Update create_user and update_user length in tables, so business bceid user's cognito_user_id can be inserted as create/update user

`fam_app_environment`, `fam_role_type` and `fam_user_type_code` has no create_user and update_user column
`fam_application_admin` no need to change, cause no bceid can grant application admin, but for consistency, updated it as well. 

Image

### CODE REVIEW

**Comment by MCatherine1994** [comment]
I wonder for the `fam_application_admin` table, there is no need to change the create/update user length, cause no bceid can grant application admin. But do we need to keep consistency for the length and update to 100 chars as well?

**Comment by basilv** [comment]
I agree there's no need to change FAM_APPLICATION, but its easier for consintency to have all these audit columns the same length, so lets ensure that audit columns in all tables are updated.

File: server/flyway/sql/V45__create_and_update_user_length_increase.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+-- Increase create_user and update_user to (100) because business bceid user's cognito_user_id is longer than 60 chars
+
+ALTER TABLE app_fam.fam_application
+    ALTER COLUMN create_user SET DATA TYPE VARCHAR(100),
+    ALTER COLUMN update_user SET DATA TYPE VARCHAR(100);
+
+ALTER TABLE app_fam.fam_application_client
```
Comment (basilv) [inline server/flyway/sql/V45__create_and_update_user_length_increase.sql:7]: Missing alter table for fam_application_admin (model was changed).

**Review by basilv — CHANGES_REQUESTED** [review]
add fam_application_admin and any other tables with audit columns to sql script

**Review by basilv — APPROVED** [review]

**Pull request merged**